pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -507,7 +514,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 			return ctrl.Result{}, nil
 		}
 
-		r.markAsFailed(msgErrorUpdateSecret, err, externalSecret, syncCallsError.With(resourceLabels))
+		if ctrlerrors.IsNamespaceGone(err) {
+			err = nil
+		} else {
+			r.markAsFailed(msgErrorUpdateSecret, err, externalSecret, syncCallsError.With(resourceLabels))
+		}
```

Comment:
This is effectively removing this ExternalSecret from ever being requeued 😄 I am pretty sure you don't want to do that.

Reply:
Nevermind me - we are already considering a deletion will come at this stage